kili/queries/label/__init__.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, auth):
     # pylint: disable=dangerous-default-value
     @Compatible(['v1', 'v2'])
     @typechecked
-    @deprecate(removed_in="2.116")
+    @deprecate(removed_in="2.118")
```

Comment:
This is tool to remember to remove the deprecation warning in the 2.116 (which is here the project_id), we should stick to `@deprecate(removed_in="2.116")` and change it to 2.118 once that we will have removed the first deprecation